sql/schema.sql

```diff
@@ -6,31 +6,31 @@
 -----------------
 
 CREATE TABLE IF NOT EXISTS repos
-( id               INT  NOT NULL
-, repo_name        TEXT NOT NULL
-, repo_desc        TEXT NOT NULL
-, repo_url         TEXT NOT NULL -- The repo html_url
-, open_issue_count INT  NOT NULL
-, fork_count       INT  NOT NULL
+( id         SERIAL       
+, repo_name  TEXT   NOT NULL
+, repo_owner TEXT   NOT NULL
+, repo_desc  TEXT
+-- The repo's html_url
+, repo_url   TEXT   NOT NULL 
 );
 
 CREATE TABLE IF NOT EXISTS issues
-( id           INT  NOT NULL
-, issue_number INT  NOT NULL
-, issue_title  TEXT NOT NULL
-, issue_body   TEXT NOT NULL
-, issue_url    TEXT NOT NULL
-, repo_id      INT  NOT NULL
+( id           SERIAL 
+, issue_number INT    NOT NULL
+, issue_title  TEXT   NOT NULL
+, issue_body   TEXT
+, issue_url    TEXT   NOT NULL
+, repo_id      INT    NOT NULL
 );
 
 CREATE TABLE IF NOT EXISTS categories
-( id            INT  NOT NULL
-, category_name TEXT NOT NULL
+( id            SERIAL
```

Comment:
Just my personal taste, by I prefer to have `PRIMARY KEY` near `SERIAL`. Constraints and foreign keys can be big and they change often, so it's better to specify them separately via `ALTER` statements. But changing the primary key will result in changing everything, so it's okay to specify it directly in the table.

Reply:
Okay cool that makes sense. Will fix it.